docs/how-to-debug-notifications.md

```diff
@@ -0,0 +1,77 @@
+# How to debug notifications
+
+## Get Firebase project id
```

Comment:
In this section I'm missing two things:
- Where should I put this code?
- Why should I do this? (i.e. is this a prerequisite do to tests? What should I do with the projectId? etc)

Reply:
I updated the documentation. Tell me if it is enough or not.